backend/oasst_backend/api/v1/tasks.py

```diff
@@ -9,12 +10,36 @@
 from oasst_backend.tree_manager import TreeManager
 from oasst_shared.exceptions import OasstError, OasstErrorCode
 from oasst_shared.schemas import protocol as protocol_schema
+from oasst_shared.schemas.protocol import User
 from sqlmodel import Session
 from starlette.status import HTTP_204_NO_CONTENT
 
 router = APIRouter()
 
 
+@dataclass
+class TaskContext:  # todo: come up with a better name?
+    pr: PromptRepository
```

Comment:
Can you spell this out completely?
```suggestion
    prompt_repository: PromptRepository
```

Reply:
Removed the context class, but `TreeManager` still uses a short `pr` property. Do you think it makes sense to update that one instead?